doc/tech/glossary.md

```diff
@@ -131,9 +131,20 @@ Terms not defined by Kubernetes:
 
 **Software Defined Storage (SDS)** architecture consists of the storage software that is independent from the underlying storage hardware. The storage access software provides data request interfaces (APIs) and the SDS controller software provides storage access services and networking.
 
-
 <a name="1.6"></a>
-## 1.6 Other Referenced Terminology
+## 1.6 Terminology For Decoupling Applications From Infrastructure, Platform and Other Application Functionality:
+- Decoupling, Loose Coupling = Loosely coupled system is one in which each of its components has, or makes use of, little or no knowledge of the definitions of other separate components. Loose coupling is the opposite of tight coupling. ([Wikipedia:Loose Coupling](https://en.wikipedia.org/wiki/Loose_coupling))
```

Comment:
@petorre I realise this is from wikipedia but can we qualify "definitions" as APIs could qualify as "definitions"?

Reply:
Changed to "implementation details".